pkg/cmd/cli/cmd/version.go

```diff
@@ -126,13 +126,14 @@ func (o VersionOptions) RunVersion() error {
 		}
 
 		ocVersionBody, err := oClient.Get().AbsPath("/version/openshift").Do().Raw()
-		if err != nil && !(kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err)) {
+		if err != nil && !(kapierrors.IsNotFound(err) || kapierrors.IsUnauthorized(err) || kapierrors.IsForbidden(err)) {
```

Comment:
No, the JSON error for some reason "found" the endpoint, but returned an empty response, that's why I check for a non-empty byte array in [L136](https://github.com/openshift/origin/pull/10093#discussion-diff-72683831R136)